apps/web/utils/ai/group/create-group.ts

```
@@ -1,41 +1,180 @@
 import { z } from "zod";
-import { chatCompletionObject } from "@/utils/llms";
+import type { gmail_v1 } from "@googleapis/gmail";
+import { chatCompletionTools } from "@/utils/llms";
 import type { Group, User } from "@prisma/client";
+import { queryBatchMessages } from "@/utils/gmail/message";
+import { UserAIFields } from "@/utils/llms/types";
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the transpilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)